### ISSUE TITLE
Chat window is dark using lite skin

### ISSUE THREAD

**Issue by CoenraadS** [donzooles/omega-for-mumble#16]
Hello

While using Omera lite, the chat Window is the same as the dark theme.

**Comment by donzooles** [comment]
Yes I did this to create a better read experience, I'll make this optional in future updates, stay tuned! :)
